android/app/build.gradle

```diff
@@ -208,6 +208,7 @@ dependencies {
     implementation project(':watermelondb')
     implementation project(':reactnativenotifications')
     implementation project(":reactnativekeyboardinput")
+    
```

Comment:
Not needed

Reply:
@diegolmello What is not needed here? Sorry.

Reply:
This empty space

Reply:
Actually it is common on Android side have a separation like this but with the `fileTree` at the top, the the `project` and finally the libraries declaration - each of them separated by empty lines. Do we have any rule on this project that block us that standard?